content/sensu-go/5.0/getting-started/platforms.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Supported platforms"
+linkTitle: "Supported Platforms"
+description: "List of supported platforms for the Sensu backend, agent, and command-line tool"
+version: "5.0"
+weight: 5
+product: "Sensu Go"
+menu:
+  sensu-go-5.0:
+    parent: getting-started
+---
+
+### Sensu backend
+
+The Sensu backend is available for 64-bit Linux.
+See the [backend installation guide][1] for more information.
+
+| Platform & Version | `amd64` |   | | |
+|--------------------|-------|-------|---|---|
+| CentOS/RHEL 5      | ✅     |      | | |
+| CentOS/RHEL 6      | ✅     |      | | |
+| CentOS/RHEL 7      | ✅     |      | | |
+| Debian 8           | ✅     |     | | |
+| Debian 9           | ✅     |     | | |
+| Ubuntu 16.04       | ✅     |      | | |
+| Ubuntu 18.04       | ✅     |      | | |
+| Ubuntu 18.10       | ✅     |      | | |
+
+### Sensu agent
+
+The Sensu agent is available for Linux and Windows.
+See the [agent installation guide][2] for more information.
+
+| Platform & Version | `amd64` |   | | |
+|--------------------|-------|-------|---|---|
+| CentOS 5/RHEL      | ✅     |    | | |
+| CentOS 6/RHEL      | ✅     |     | | |
+| CentOS 7/RHEL      | ✅     |     | | |
+| Ubuntu 16.04       | ✅     |     | | |
+| Ubuntu 18.04       | ✅     |     | | |
+| Ubuntu 18.10       | ✅     |     | | |
+| Debian 8           | ✅     |     | | |
+| Debian 9           | ✅     |     | | |
+| Windows 7 and later| ✅     |     | | |

```

Comment:
I think we should leave `Windows 7 and later` out of the supported OS's for the agent. We haven't done much (or any) testing of the agent on Windows 7. I'm also not sure if anyone will want to run the agent on a desktop OS.

It does make sense to include it for the `sensuctl` tool though.

Thoughts @annaplotkin?

Reply:
+1 on sensuctl and leaving it out of agent 👍 @amdprophet

Reply:
Updated!